documentation/modules/ref-kafka-bridge-http-configuration.adoc

```diff
@@ -20,6 +22,12 @@ metadata:
 spec:
   # ...
   http:
-    port: 8080
+    port: 8080 <1>
+    cors:
+      allowedOrigins: "https://strimzi.io" <2>
+      allowedMethods: "GET,POST,PUT,DELETE,OPTIONS,PATCH" <3>
   # ...
 ----
+<1> The default HTTP configuration for the Kafka Bridge to listen on port 8080.
+<2> Comma-separated list of allowed CORS origins. You can use a URL or a Java regular expression.
+<3> Comma-separated list of allowed CORS HTTP methods.
```

Comment:
Are these "CORS HTTP methods" or plain old HTTP methods?

Reply:
Better to say _allowed HTTP methods for CORS_